apps/bestofjs-nextjs/src/server/api-hall-of-fame.tsx

```diff
@@ -38,3 +64,15 @@ function fetchHallOfFameData() {
     heroes: BestOfJS.RawHallOfFameMember[];
   }>;
 }
+
+function filterMembers(members: BestOfJS.RawHallOfFameMember[], query: string) {
+  const criteria = {
+    $or: [
+      { name: { $regex: new RegExp(query, "i") } },
```

Comment:
Its a bit difficult to read this part for me. Maybe I'm just not used to mingo yet.

Do you think we can simplify it like this?

```
const regex = new RegExp(query, "i");

  const criteria = {
    $or: [
      { name: { $regex: regex} },
      { username: { $regex: regex} },
    ],
  };

```

Reply:
Or

```
const regex = new RegExp(query, "i");
const mingoRegex = { $regex: regex };
  const criteria = {
    $or: [
      { name: { $regex: regex } },
      { username: { $regex: regex } },
    ],
  };
```